src/index.html

```diff
@@ -7,8 +7,54 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous"
+    >
```

Comment:
please fix the code styling according to the checklist. fix all places in HTML file
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/91ab0652-afba-4738-9293-883ecda139cf)